Sources/VLCHTTPUploaderController.h

```diff
@@ -15,11 +15,14 @@
 
 #import <UIKit/UIKit.h>
 
+@class MediaLibraryService;
+
 @interface VLCHTTPUploaderController : NSObject
 
 + (instancetype)sharedInstance;
 
 @property (nonatomic, readonly) BOOL isReachable;
+@property (nonatomic, readwrite) MediaLibraryService *medialibrary;
```

Comment:
Perhaps this could be a task for a follow-up ticket to have the service injected, and remove the usage of the shared instance. 👍

Reply:
yeah that really should be a follow up the shared instance angered me a lot here